tests/unit/states/test_ansiblegate.py

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2020 SUSE LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Import Salt Testing Libs
+
+import json
+import os
+
+import salt.states.ansiblegate as ansible
+import salt.utils.files
+import salt.utils.platform
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.runtests import RUNTIME_VARS
+from tests.support.unit import TestCase, skipIf
+
+try:
+    import pytest
+except ImportError as import_error:
+    pytest = None
+NO_PYTEST = not bool(pytest)
+
+
+@skipIf(NO_PYTEST, False)
```

Comment:
Just cosmetic request to drop the redundant `NO_PYTEST` constant. Other tests are using `pytest` directly in `skipIf`
```python
@skipIf(pytest is None, "PyTest is missing")
```

Reply:
:+1: